content/en/Features/networking/network_monitoring.md

```diff
@@ -0,0 +1,31 @@
+---
+title: "Network Monitoring"
+linkTitle: "Network Monitoring"
+date: 2022-03-11
+weight: 3
+draft: false
+description: >
+  How to monitor the network
+---
+
+### Cloud provider network networking
+
+Each cloud provider has some preconfigured service to monitor networking such as [Amazon CloudWatch](https://aws.amazon.com/cloudwatch/), [Google operations suite](https://cloud.google.com/products/operations) and [Azure Monitor](https://azure.microsoft.com/en-us/services/monitor/).
+
+### Linkerd Viz
```

Comment:
We have a tutorial for this somewhere as well. Let's make sure we don't duplicate this

Reply:
github doesn't show the diff on this but this section is moved from `content/en/Features/service2service.md`  (it shows as deleted in the PR)